clippy.toml

```diff
@@ -1 +1,2 @@
 avoid-breaking-exported-api = false
+doc-valid-idents = ["OpenAPI", ".."]
```

Comment:
Could you elaborate (and maybe add as a doc-string) on why this is necessary?

Reply:
[b0b7de2](https://github.com/geo-engine/geoengine/pull/760/commits/b0b7de2d20575d232ef0cbcb4b139b4600770743)